lib/wallet/channel_change_notifier.dart

```diff
@@ -2,15 +2,87 @@ import 'dart:async';
 
 import 'package:flutter/material.dart';
 import 'package:ten_ten_one/bridge_generated/bridge_definitions.dart';
+import 'package:ten_ten_one/cfd_trading/validation_error.dart';
 import 'package:ten_ten_one/ffi.io.dart' if (dart.library.html) 'ffi.web.dart';
+import 'package:ten_ten_one/wallet/action_card.dart';
+import 'package:ten_ten_one/wallet/open_channel.dart';
 
 class ChannelChangeNotifier extends ChangeNotifier {
-  ChannelState state = ChannelState.Unavailable;
+  ChannelState? state;
 
   bool isInitialising() => state == ChannelState.Establishing;
 
+  bool isDisconnected() => state == ChannelState.Disconnected;
+
   bool isAvailable() => state == ChannelState.Available;
 
+  bool isUnavailable() => state == ChannelState.Unavailable;
+
+  Message? validate() {
```

Comment:
The word `validate` is quite vague in this context.

```suggestion
  Message? statusMessage() {
```